lib/path/path.cpp

```diff
@@ -1,19 +1,19 @@
-#include "path.h"
+#include "path.hpp"
 
-#include "platform.h"
+#include "platform.hpp"
 #include "utils.h"
 
-inline const path_pose_t *path_get_current_pose(const path_t *path)
+/* inline */ const path_pose_t *path_get_current_pose(const path_t *path)
```

Comment:
function cannot be inline in c++ ?
if not remove it

Reply:
Ok, I left it in comment to discuss about that.
It's almost the same in C and C++, except that in C++, inline functions are not generated, so these functions can't be used in other files.
Although, in C and C++, inlining is only effective in the same file.
If we want some functions to be inlined in other files, we have to define them in the headers.